{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/components/admin.py

```diff
@@ -7,3 +7,8 @@
 @admin.register(CallToAction)
 class CallToActionAdmin(admin.ModelAdmin{% if cookiecutter.sections == 'yes' %}, UsedOnAdminMixin{% endif %}):
     list_display = ['title'{% if cookiecutter.sections == 'yes' %}, 'pages_used_on'{% endif %}]
+    fieldsets = (
```

Comment:
Not that it makes much difference, but FYI you can this here instead:

```python
fields = ['kicker', 'title', 'link_text', 'link_page', 'link_url']
```

Reply:
It's just my preference to have `fieldsets` in `admin.py`s and `fields` in `forms.py`s

Reply:
I'm OK with that. :)